ArduCopter/Copter.cpp

```diff
@@ -347,16 +347,24 @@ bool Copter::set_target_posvelaccel_NED(const Vector3f& target_pos, const Vector
     return mode_guided.set_destination_posvelaccel(pos_neu_cm, vel_neu_cms, accel_neu_cms, use_yaw, yaw_deg * 100.0, use_yaw_rate, yaw_rate_degs * 100.0, yaw_relative);
 }
 
-bool Copter::set_target_velocity_NED(const Vector3f& vel_ned)
+bool Copter::set_target_velocity_NED(const Vector3f& vel_ned, bool align_yaw_to_target)
```

Comment:
It would definitely be nice to be able to control the yaw better from scripting

won't this break the API though for existing users?